config/nrfconnect/chip-module/Kconfig

```diff
@@ -493,4 +493,11 @@ config CHIP_IM_PRETTY_PRINT
 	  Request). This option helps in debugging and development of message exchanges within the Matter
 	  protocol.
 
+config CHIP_CRYPTO_USE_X509
+	bool "MbedTLS X.509 certificate support"
```

Comment:
Do we need to mention MbedTLS or rather say "x.509 certificate support" and instead do depends MBEDTLS or CHIP_CRYPTO_MBEDTLS (not sure which if any is correct)?